ingest_python_scripts/smithsonian/launch_smithsonian.py

```
@@ -116,7 +116,19 @@ def ssm_run(shell_cmd, timeout_seconds=30, poll_seconds=4):
     ])
     if status != "Success":
-        raise RuntimeError(f"SSM ended with status {status}.\nOutput:\n{output}")
+        try:
+            stderr = _aws([
+                "ssm", "get-command-invocation",
+                "--command-id", cmd_id,
+                "--instance-id", INSTANCE_ID,
+                "--query", "StandardErrorContent",
+                "--output", "text",
+            ])
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)